sdk/react/widgets/SkipButton.js

```diff
@@ -37,6 +38,9 @@ class SkipButton extends Component {
   }
 
   render() {
+    if(!this.props.visible) {
```

Comment:
Add space `if (`

Reply:
We will fix it during the process of lint errors fixing.

Reply:
@maxkupetskii this is a job for eslint, not for developer